### PR TITLE
Added Monitor and Human Interaction pattern samples in JavaScript,

### PR DESCRIPTION
**Pull request by kashimiz**
clarified required TwilioPhoneNumber app setting format. Monitor sample
requires a build of Microsoft.Azure.WebJobs.Extensions.DurableTask that
includes change c29c0e8bfd97b5ea68689a7ef6de1ff14fe1a25b to work.

Step one of #329 .

### CODE REVIEW

File: samples/javascript/E3_Monitor/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+module.exports = df(function*(context) {
+    const input = context.df.getInput();
+    context.log("Received monitor request. location: " + (input ? input.location : undefined)
+        + ". phone: " + (input ? input.phone : undefined) + ".");
+
+    verifyRequest(input);
+
+    const endTime = moment.utc(context.df.currentUtcDateTime).add(6, 'h');
+    context.log("Instantiating monitor for " + input.location.city + ", " + input.location.state
+        + ". Expires: " + (endTime) + "."); // tostring in JavaScript?
```
Comment (cgillum) [inline samples/javascript/E3_Monitor/index.js]: Do we want this comment in the official samples?

File: samples/javascript/extensions.csproj [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+	<WarningsAsErrors></WarningsAsErrors>
+	<DefaultItemExcludes>**</DefaultItemExcludes>
```
Comment (cgillum) [inline samples/javascript/extensions.csproj]: Looks like we have a mix of spaces and tabs. Can you convert this to all spaces?

File: samples/javascript/extensions.csproj [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.webJobs.Extensions.DurableTask" Version="1.4.1" />
```
Comment (cgillum) [inline samples/javascript/extensions.csproj]: This version of the extension doesn't include the required changes, does it? Should we increment the extension version as part of this PR?

File: samples/javascript/local.settings.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
     "AZURE_STORAGE_CONNECTION_STRING": "<Put your Azure Storage connection string here>",
     "TwilioAccountSid": "<Put your Twilio Account SID here>",
     "TwilioAuthToken": "<Put your Twilio Auth Token here>",
-    "TwilioPhoneNumber": "<Put your Twilio Phone Number here>",
+    "TwilioPhoneNumber": "<Put your Twilio Phone Number here in E.164 format>",
```
Comment (cgillum) [inline samples/javascript/local.settings.json]: Should we add this update to the other samples as well?

Comment (kashimiz) [reply samples/javascript/local.settings.json]: Yes, agreed. I'll submit this as a separate PR.

File: samples/javascript/extensions.csproj (hunk 3 above)
Comment (kashimiz) [reply samples/javascript/extensions.csproj]: Oh! Yes, if we can. That would be great.

File: samples/javascript/extensions.csproj (hunk 2 above)
Comment (kashimiz) [reply samples/javascript/extensions.csproj]: Not sure how that happened; will do.

File: samples/javascript/E3_Monitor/index.js (hunk 1 above)
Comment (kashimiz) [reply samples/javascript/E3_Monitor/index.js]: No we don't. Good catch.

File: src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj [hunk 5]
```diff
@@ -5,9 +5,9 @@
     <AssemblyName>Microsoft.Azure.WebJobs.Extensions.DurableTask</AssemblyName>
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.DurableTask</RootNamespace>
     <DocumentationFile>Microsoft.Azure.WebJobs.Extensions.DurableTask.xml</DocumentationFile>
-    <AssemblyVersion>1.4.1.0</AssemblyVersion>
-    <FileVersion>1.4.1.0</FileVersion>
-    <Version>1.4.1</Version>
+    <AssemblyVersion>1.5.0.0</AssemblyVersion>
+    <FileVersion>1.5.0.0</FileVersion>
+    <Version>1.5.0</Version>
```
Comment (cgillum) [inline src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj]: Is your PR sync'd to latest? This will conflict with my existing change.

**Pull request merged**